tests/test_bincopy.py

```diff
@@ -569,40 +569,69 @@ def test_performance(self):
         binfile = bincopy.BinFile()
         binfile.add_srec(srec)
 
-    def test_command_line_info_help(self):
-        argv = ['bincopy', 'info', '--help']
-        output = """usage: bincopy info [-h] binfile [binfile ...]
+    def test_command_line_help(self):
+        commands_descriptions=[
+            ('info','Print general information about given file(s).'),
+            ('as_hexdump','Print hexdump to stdout.'),
+            ('as_srec','Print srec to stdout.'),
+            ('as_ihex','Print ihex to stdout.')]
 
-Print general information about given file(s).
+        for tupl in commands_descriptions:
```

Comment:
Replace with `for name, description in commands_descriptions:`, or similar. An unpacked tuple is much easier to understand than accessing its items by index.